### PR TITLE
Add mappings for new column types

### PR DESCRIPTION
**Pull request by ekoleda-codaio**
Add mappings for Link, Canvas, and Email column types.

### CODE REVIEW

**Review by patrick-codaio — APPROVED** [review]

File: docs/guides/basics/data-types.md [hunk 1]
```diff
@@ -315,29 +315,32 @@ The [`Reference`][Reference] value hint can be used to reference a row in a sync
...
+| Currency      | ✅ Yes    | `Number`             | `Currency`        |
+| Slider        | ✅ Yes    | `Number`             | `Slider`          |
+| Scale         | ✅ Yes    | `Number`             | `Scale`           |
+| Date          | ✅ Yes    | `String` or `Number` | `Date`            |
+| Time          | ✅ Yes    | `String` or `Number` | `Time`            |
+| Date and time | ✅ Yes    | `String` or `Number` | `DateTime`        |
+| Duration      | ✅ Yes    | `String`             | `Duration`        |
+| Checkbox      | ✅ Yes    | `Boolean`            |                   |
+| People        | ✅ Yes    | `Object`             | `Person`          |
+| Email         | ❌ No     |                      |                   |
```
Comment (spencer-codaio) [inline docs/guides/basics/data-types.md:337]: just curious are we planning on supporting this? And is the difference between this section and the "parameters" section that this section is talking about what types we support as result types?

**Review by spencer-codaio — APPROVED** [review]

File: docs/guides/basics/data-types.md (hunk 1 above)
Comment (ekoleda-codaio) [reply docs/guides/basics/data-types.md:337]: I don't know if anyone is working on it, but ideally we would support it. This section is what value hints you can apply to your formula or schema property to get the equivalent column type in Coda. The parameters version is about what parameter type we recommend you use to pass that value to your formula.

Comment (spencerc99) [reply docs/guides/basics/data-types.md:337]: Ahhh gotcha thank you!

Comment (spencerc99) [reply docs/guides/basics/data-types.md:337]: ahh gotcha thank you!

Comment (jonathan-codaio) [reply docs/guides/basics/data-types.md:337]: Oh Mike has PRs out right now to add support for this.

Comment (ekoleda-codaio) [reply docs/guides/basics/data-types.md:337]: Sweet! Will update this table once that PR is in.

**Pull request merged**